src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/QueryIT.java

```diff
@@ -971,16 +1032,16 @@ public void useScrollWithOrderByAndParams() throws IOException {
         }
     }
 
-    @Ignore // Subquery is still running in ES transport thread which fails the thread assertion as well as tests followed.
+    @Ignore // Subquery still runs in ES transport thread which fails the thread assertion as well as tests followed.
```

Comment:
NP: Should the ending of this comment be reworded to something like ```as well as the following tests```?

Reply:
reworded